nnstreamer.ini.in

```diff
@@ -8,6 +8,9 @@ enable_symlink=@ENABLE_SYMBOLIC_LINK@
 filters=@SUBPLUGIN_INSTALL_PREFIX@/filters/
 customfilters=@SUBPLUGIN_INSTALL_PREFIX@/customfilters/
 
+# Set framework priority about model file extension when automatically selecting framework for tensor filter.
```

Comment:
I think this needs more explanations because the comment is not intuitive (e.g., too generic description).
For example, users may see only this `.ini` file, not build-related files (e.g., `meson_options.txt`, `meson.build`).

I think the below comment in `meson_options.txt` is better. So, please reuse this.
```
option(..., description: 'A comma separated prioritized list of neural network frameworks to open .tflite file')
```

Reply:
Agree! I will update explanation.